src/raftstore/store/snap.rs

```diff
@@ -650,7 +655,11 @@ impl Snap {
                 }
             }
             let size = get_file_size(&cf_file.tmp_path)?;
-            if size > 0 {
+            // The size of a sst file is larger than 0 doesn't mean it contains some key value pairs.
+            // For example, if we provide a encrypted env to the SstFileWriter, RocksDB will append
```

Comment:
Does it matter when rolling update new TiKV sends encrypted snapshots to TiKV without encryption enabled?

Reply:
This is forbidden. This feature is design for new cluster, and doesn't support turn on/off for existing cluster.